tests/baselines/reference/lambdaParameterWithTupleArgsHasCorrectAssignability.types

```diff
@@ -33,7 +33,7 @@ function consumeClass(c: GenericClass<[string, boolean]>) { }
 consumeClass(createClass(str => console.log(str.length)));
 >consumeClass(createClass(str => console.log(str.length))) : void
 >consumeClass : (c: GenericClass<[string, boolean]>) => void
->createClass(str => console.log(str.length)) : GenericClass<[str: string]>
+>createClass(str => console.log(str.length)) : GenericClass<[string, boolean]>

```

Comment:
This also looks correct in context; this is the same type from the previous line.